src/main/java/pl/miloszkajetan/serwisAukcyjny/user/UserRegistrationBuilder.java

```diff
@@ -0,0 +1,16 @@
+package pl.miloszkajetan.serwisAukcyjny.user;
+
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+public class UserRegistrationBuilder {
+
+    public static User userRegistration (UserRegistration userRegistration, PasswordEncoder passwordEncoder){
+
+        User user = new User();
+        user.set // Builder nie działa :(
```

Comment:
W jaki sposób nie działa? 
Jeżeli dajesz adnotacje Builder (słusznie) to robisz 
User.builder()
.nazwaPola(wartość)
.nazwaPola(wartość)
.build()